2019-04-08 22:34:45,768 - root - ERROR - get请求出错，出错原因:HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /todos?not%20argument (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000239D28812B0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',))
2019-04-08 22:34:45,768 - root - INFO - 请求传入数据：url:http://127.0.0.1:5000/todos,key:1.0,参数:not argument,请求方式：GET
2019-04-08 22:34:45,768 - root - INFO - 测试用例执行完毕
2019-04-08 22:34:45,769 - root - INFO - ----------------------------------
2019-04-08 22:34:45,786 - root - INFO - 测试用例开始执行
